CHANGELOG.rst

```diff
@@ -3,6 +3,9 @@
 0.65.0
 ------
 
+0.64.1
```

Comment:
I think it makes sense to add a single line about this change in the changelog.

Reply:
Will do!